web/src/bundles/app.ts

```diff
@@ -14,6 +14,7 @@ import "../templates";
 import "../zulip_test";
 
 // Import styles
+import "tippy.js/dist/border.css";
```

Comment:
Should we have a comment here documenting what these modules do, potentially just in forms to the right upstream links. https://atomiks.github.io/tippyjs/v5/themes/ does not make it obvious what this particular file does.

Reply:
I found it in the `v6` docs: https://atomiks.github.io/tippyjs/v6/themes/#arrow-border

Reply:
Should we still consider adding a comment linking to the above URL in the tippyjs doc?

Reply:
@timabbott a gentle ping towards this.

Reply:
I think it'd be helpful if you split this first commit into two commits -- a first one that adds this and refactors the actual CSS, and a second one that just renames things.

It's really really hard to review commits that do both at the same time, and I think the actual CSS changes are small enough that it should be easy to extract.

Reply:
I understand. I have the split the first commit into two smaller commits as suggested. One focusing on the renames, the other on the tippy arrows for the popovers.

Reply:
I have also added the reference to https://atomiks.github.io/tippyjs/v6/themes/#arrow-border, along with a small accompanying comment.